### ISSUE TITLE
input with tag passes arguments in lowercase

### ISSUE THREAD

**Issue by HomeSeer** [lunet-io/scriban#168]
Normally, strings passed to functions are passed as is. But in the markup below the passed string is converted to lower case. Not sure why. The workaround is to call the function outside the markup and then just insert the variable. But it would be ideal if this worked. The parameter below "gEnableEventProcessing" gets passed to my .net class as lower  case.

`
<input type="checkbox" class="form-check-input" id="gEnableEventProcessing" name="gEnableEventProcessing" {{ get_setting_bool 'gEnableEventProcessing' }} >
`

**Comment by xoofx** [comment]
Difficult to say, I don't recall that we are doing something in particular... you example is a bit weird because you have nested `{{ get_setting_bool`  which would not work, specially the first string after `get_setting_bool ` is going to eat until the next `'`

**Comment by HomeSeer** [comment]
Thanks for commenting, I see that my post was messed up, copy/paste error. I fixed the html in my original post. To get it to work I did:

{{ chk=get_setting_bool 'gEnableEventProcessing' }}

Then use {{ chk  }} in place of the call. This works ok. So very odd. The call is made, but the parameter is converted to lowercase. I checked the HTML that was generated and the parameter is ok, so its converted only when the call is made and the parameter is passed.
